api/compression.md

```diff
@@ -11,14 +11,36 @@ policy](/compression/add_compression_policy/) for when to compress chunks.
 Advanced usage of compression allows users to [compress chunks
 manually](/compression/compress_chunk), instead of automatically as they age.
 
+Compressed chunks have the following limitations:
+  - Constraints are restricted to compression `SEGMENTBY` columns when
+    applicable. Otherwise they are unsupported.
+  - `ROW LEVEL SECURITY` is not supported.
+
```

Comment:
This should be added here and not under 2.3  (see my comment below for line 42 onwards)

>     - Unique constraints (`UNIQUE`) are restricted to compression `SEGMENTBY`
>       and `ORDER BY` columns.
>     - Primary Keys (`PRIMARY KEY`) are restricted to compression `SEGMENTBY`
>       and `ORDER BY` columns.